gravitee-gateway-http/src/main/java/io/gravitee/gateway/http/connector/VertxProxyConnection.java

```diff
@@ -94,6 +95,15 @@ public void handleConnectTimeout(Throwable throwable) {
     @Override
     public VertxProxyConnection write(Buffer chunk) {
```

Comment:
I can't ensure that deffering the write of headers wouldn't cause any regression though. (hope tests would catch any). but lgtm